### ISSUE TITLE
Question about image resizing and evaluation methods in localization task

### ISSUE THREAD

**Issue by shot1107** [gudovskiy/cflow-ad#3]
I'm a student who studies anomaly detection, Thank you for providing an interesting paper and well-designed code.
I would like to ask a question about image resizing and evaluation methods in the localization task.

The input image is resized when training in your code. And the information about the size of the resized image is given when the main.py is executed like this.
`python3 main.py --gpu 0 --pro -inp 128 --dataset mvtec --class-name transistor`
The original size of "transistor" class in MVTec dataset is 1024, so I guess the size of this image is resized from 1024 to 128 here.

My question:
Are the predicted mask image and GT mask image also resized when they are tested? (like 1024 -> 128 in "transistor" class)
If so, would reducing the resolution of the image used for testing not improve the evaluation value?

I'd appreciate it if you could answer my questions.

**Comment by gudovskiy** [comment]
@shot1107 yes, they are both resized to [c.crp_size](https://github.com/gudovskiy/cflow-ad/blob/b50828b4536a28b7b32f93cb67691dedf0dae100/train.py#L321) (e.g. 128x128 for transistor)
Probably, reducing ground truth mask resolution can decrease eval values a bit. I didn't really check the difference.
On the other hand, the predicted masks are at even lower resolution (from 64x64 to 8x8), which can be the real eval bottleneck. So, selection of these hyperparameters can significantly affect accuracy/speed tradeoff.

**Comment by shot1107** [comment]
If reducing ground truth mask resolution can improve eval values, I thought it's not good in terms of the development of anomaly detection method. But according to your comment, that doesn't seem to be a problem.
I will do some experiments to confirm it.
Anyway, thank you for replying :)
